wdl-lsp/tests/common.rs

```diff
@@ -0,0 +1,281 @@
+//! Common test suite for WDL LSP integration tests.
+
+use std::fmt::Debug;
+use std::fs;
+use std::io;
+use std::path::Path;
+use std::path::PathBuf;
+
+use tempfile::TempDir;
+use tokio::io::AsyncBufReadExt;
+use tokio::io::AsyncReadExt;
+use tokio::io::AsyncWriteExt;
+use tokio::io::BufReader;
+use tokio::io::DuplexStream;
+use tokio::io::duplex;
+use tower_lsp::LspService;
+use tower_lsp::jsonrpc;
+use tower_lsp::lsp_types;
+use tower_lsp::lsp_types::ClientCapabilities;
+use tower_lsp::lsp_types::InitializeParams;
+use tower_lsp::lsp_types::InitializedParams;
+use tower_lsp::lsp_types::WorkspaceDiagnosticParams;
+use tower_lsp::lsp_types::WorkspaceFolder;
+use tower_lsp::lsp_types::notification::Notification;
+use tower_lsp::lsp_types::request::Request;
+use tower_lsp::lsp_types::request::WorkspaceDiagnosticRequest;
+use url::Url;
+use wdl_lsp::Server;
+use wdl_lsp::ServerOptions;
+
+/// Copied from https://stackoverflow.com/a/65192210
```

Comment:
Let's use this instead: https://docs.rs/fs_extra/latest/fs_extra/fn.copy_items.html. I just don't love the idea of copying stackoverflow code without any indication of a license.

Reply:
I was using `fs_extra` earlier. 

This is copied from `wdl-doc/tests` which is copied from SO
https://github.com/stjude-rust-labs/wdl/blob/368c81dec639334af2ea638338100a0bb7f84c4a/wdl-doc/tests/doc.rs#L18-L21

Reply:
Does `fs_extra` work? If so, I'd actually rather replace both instances of this function with that.